appveyor.yml

```diff
@@ -23,7 +23,7 @@ install:
   - "python.exe -m pip install wheel"
   - "python.exe -m pip install unittest-xml-reporting"
   # Use last binary builds, see https://github.com/PyMySQL/mysqlclient-python/issues/262
-  - "python.exe -m pip install mysqlclient==1.3.13"
+  - "python.exe -m pip install mysqlclient==1.3.12"
```

Comment:
Looks like wrong rebase, this is already part of master.

Reply:
Yes, I picked an old commit. I tought it would stay untouched.. Bad guess!